### DISCUSSION TITLE
Add icon to Grafana UI library

### DISCUSSION THREAD

**Discussion by mary-sitzenstatter** [grafana/grafana#43687] [Platform]
I am working on Recorded queries with @MasslessParticle @toddtreece @bossinc - we would like to use a "record" icon for this new feature - but we don't have this available in the Grafana UI library. 

I'm wondering what steps we have to take to get a "record" icon added - we found a few good options on iconscout:

https://iconscout.com/icon/vinyl-record-2848409
https://iconscout.com/icon/record-1440310

Thanks.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
